patches/chrome-browser-language-android-java-src-org-chromium-chrome-browser-language-settings-LanguageSettings.java.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/language/android/java/src/org/chromium/chrome/browser/language/settings/LanguageSettings.java b/chrome/browser/language/android/java/src/org/chromium/chrome/browser/language/settings/LanguageSettings.java
+index 2de272e8e851996eeb823a09fc650237cd169c02..f81c3c5034b389ba09398c2590786587dd7dd113 100644
+--- a/chrome/browser/language/android/java/src/org/chromium/chrome/browser/language/settings/LanguageSettings.java
++++ b/chrome/browser/language/android/java/src/org/chromium/chrome/browser/language/settings/LanguageSettings.java
+@@ -37,7 +37,7 @@ import java.util.Collection;
+  * Settings fragment that displays information about Chrome languages, which allow users to
+  * seamlessly find and manage their languages preferences across platforms.
+  */
+-public class LanguageSettings extends PreferenceFragmentCompat
```

Comment:
can this be converted to asm?

Reply:
This patch was removed later https://github.com/brave/brave-core/pull/8307/commits/502aa178d5b852c539664c7bf79a667e2775f0f6